manifests/s/Stream-Pi/Server/1.0/Stream-Pi.Server.installer.yaml

```diff
@@ -0,0 +1,21 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Stream-Pi.Server
+PackageVersion: 1.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+#  Scope: 
```

Comment:
```suggestion
  Scope: machine
```

Reply:
The Scope should be mandatory tbh, as voted here https://github.com/microsoft/winget-pkgs/discussions/16219#discussioncomment-827007